gallery/workbook/microsoft.network-azurefirewalls.json

```diff
@@ -15,8 +15,13 @@
 					"id": "Workbooks/Network Insights/AzureFirewallWorkbooks/NetworkInsights-AzureFirewallMetrics",
 					"author": "Microsoft",
 					"name": "Network Insights AzureFirewalls"
+				},
+				{
+					"id": "Workbooks/Network Insights/AzureFirewallWorkbooks/NetworkInsights-AzureFirewallMetrics-StructuredLogs",
+					"author": "Microsoft",
+					"name": "Network Insights AzureFirewalls"
```

Comment:
this has the exact same name as the template above it?  the build should be failing that too if it isn't already.  names need to be unique.

Reply:
and it is unclear to my why the names are so long?
the name of the group/category in the gallery is already "Network Insights AzureFirewalls" which isn't super useful, and then the items inside all start with "Network Insights AzureFirewalls" again?

these are folders/names that users see, why not just

"Network Insights" as the category and "Firewall Metrics" and "Structured Logs", etc. user friendly names.

Reply:
I just made the changes on the folder and file names. I also fixed the single name of the template.